cloud-chatops/cron-clone.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+# When copying the contents of this file, remember to name the new file using only valid characters [a-zA-Z0-9_-]
+# Fullstops '.' and file extenstions are not permitted
+
+set -ex
+
+rm -rfv $HOME/cloud-docker-images
```

Comment:
Bit concerned that we will be deleting the folders on every run of the script.
@DavidFair - can you suggest a better way or if this is the only option

Reply:
I think I actually copied `rm -rfv $HOME/...` from one of David's old scripts so I would too be interested to see what he thinks about it working here. It does seem a little destructive but I had a chat with Kalibh last week he suggested adding the absolute path. I just noticed it should be rm `-rfv $HOME/cloud-docker-images/cloud-chatops` and not `rm -rfv $HOME/cloud-docker-images`

Reply:
In this case I think we can simply
```
git fetch  && git clean -fxd && git reset --hard origin/main
```

This should
- Get the latest server copy
- Remove any files not in the source repo (e.g. one has been removed upstream)
- Reset the files back to the original copy

That way we're not doing a full clone each time, just getting the changes / deltas between?

Reply:
Right gotcha, that sounds a little less destructive, I like it! Thanks